test/EndToEnd/Tests/Framework/AppFixture.cs

```diff
@@ -32,9 +32,12 @@ public AppFixture()
                 case "local":
                     testEnvironment = new LocalTestEnvironment(logger, containerRegistry, containerTag);
                     break;
+                case "funccapps":
```

Comment:
can we add a comment next to this case statement, mentioning the full form of funccapps => functions container apps.